### PR TITLE
SDK-965: Add README

### PR DESCRIPTION
**Pull request by davidgrayston**
- Back-ported from #39

### CODE REVIEW

File: yoti/Readme.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+3) [Installing the SDK](#installing-the-sdk) -
+How to install our SDK
+
+4) [Plugin Setup](#plugin-setup)-
+How to set up the plugin in Drupal
+
+5) [Linking existing accounts to use Yoti authentication
+](#linking-existing-accounts-to-use-yoti-authentication)
+
```
Comment (echarrod) [inline yoti/Readme.md:24]: Just the same section link here that I mentioned in https://github.com/getyoti/yoti-drupal/pull/38

File: yoti/Readme.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+## An Architectural view
+
+Before you start your integration, here is a bit of background on how the
+integration works. To integrate your application with Yoti, your back-end
+must expose a GET endpoint that Yoti will use to forward tokens.
+
+The endpoint can be configured in the Yoti Dashboard when you create/update
+your application. For more information on how to create an application please
+check our [developer
+page](https://www.yoti.com/developers/documentation/#login-button-setup).
```
Comment (echarrod) [inline yoti/Readme.md]: URL change I mentioned in https://github.com/getyoti/yoti-drupal/pull/39 👍

**Review by echarrod — APPROVED** [review]

File: yoti/Readme.md (hunk 2 above)
Comment (davidgrayston) [reply yoti/Readme.md]: Fixed in fd1ea74

File: yoti/Readme.md (hunk 1 above)
Comment (davidgrayston) [reply yoti/Readme.md:24]: Fixed in fd1ea74

**Pull request merged**